assets/js/colorpicker2/ColorPicker2.js

```diff
@@ -1,184 +0,0 @@
-// ===================================================================
```

Comment:
@Jdew192837  is this used anywhere else in the code base? You need to check this before you delete it.

Reply:
Nope it was only linked as a script for those two forms. So I just removed it. :) @teryhill

Reply:
Good enough